### PR TITLE
Fix: Lottie block issues

### PR DESCRIPTION
**Pull request by supriyajjadhav**
[Trello](https://trello.com/c/7EHNso8e/317-lottie-block-issues-present-in-master-branch)

### Description
Fix Lottie Issues added description in the card

### Types of changes
- non-breakable changes

### How has this been tested?
Tested whether it is working fine or not

### Checklist:
- [x] My code is tested
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: admin/assets/admin-menu-settings.js [hunk 1]
```diff
@@ -255,7 +255,9 @@
 				type: "POST",
 				data: data,
 				success: function(data){
-
+					if(id === 'how-to'){
```
Comment (RajkiranBagal) [inline admin/assets/admin-menu-settings.js:258]: Is this change is needed here?

File: classes/class-uagb-admin.php [hunk 2]
```diff
@@ -318,6 +318,11 @@ public static function activate_widget() {
 			$blocks[ $block_id ] = $block_id;
 			$blocks              = array_map( 'esc_attr', $blocks );
 
+			if ( 'how-to' === $block_id && 'disabled' === $blocks['info-box'] ) {
```
Comment (RajkiranBagal) [inline classes/class-uagb-admin.php:321]: Is this change is needed here?

File: classes/class-uagb-admin.php [hunk 3]
```diff
@@ -337,6 +342,11 @@ public static function deactivate_widget() {
 			$blocks[ $block_id ] = 'disabled';
 			$blocks              = array_map( 'esc_attr', $blocks );
 
+			if ( 'info-box' === $block_id && 'how-to' === $blocks['how-to'] ) {
```
Comment (RajkiranBagal) [inline classes/class-uagb-admin.php:345]: Is this change is needed here?

File: classes/class-uagb-config.php [hunk 4]
```diff
@@ -1345,6 +1345,7 @@ public static function get_block_attributes() {
 						'title'       => __( 'Button', 'ultimate-addons-for-gutenberg' ),
 						'description' => __( 'This block allows you to style button.', 'ultimate-addons-for-gutenberg' ),
 						'default'     => true,
+						'is_child'    => true,
```
Comment (RajkiranBagal) [inline classes/class-uagb-config.php:1348]: These file changes not needed here

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: admin/assets/admin-menu-settings.js (hunk 1 above)
Comment (supriyajjadhav) [reply admin/assets/admin-menu-settings.js]: Next-release changes merged in this branch... Removed everything

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**